### PR TITLE
Update version: RussellBanks.Komac.Nightly version 2.0.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132858)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by Trenly** [comment]
@russellbanks - these changes look good to you?

**Comment by russellbanks** [comment]
The installer prompts when it's ran to install in user or machine scope. I imagine this means it will need `/CURRENTUSER` or `/ALLUSERS` custom switches respectively.

**Comment by Exorcism0666** [comment]
@russellbanks error hash

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 1/6/2024 8:49:49 PM
 Installer Verification Analysis Context Information: 
 Installer https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
 Scan request id: '61bad931-bb8f-489b-b23a-b4f12b7ec53a'
 

(Automated response - build 533.)

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x64-nightly.exe
-  InstallerSha256: c97a9e38345798d522ac4b86f705dfa7d754eb7a74246f29bf03d60b12b3d67c
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: d79d0c766bad4b87ee029e13ba351565e6b1fe362519a8ba4fae4c4d02ab07bb
+  InstallerSwitches:
+    Custom: /CURRENTUSER
 - Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: d79d0c766bad4b87ee029e13ba351565e6b1fe362519a8ba4fae4c4d02ab07bb
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:23]:
```suggestion
  InstallerSha256: 2AAE771EFF66B784F0A28DB99A11264040E219B3CAA64ABF19DAD5ED7523A9AD
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
  InstallerSha256: 2AAE771EFF66B784F0A28DB99A11264040E219B3CAA64ABF19DAD5ED7523A9AD
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 2aae771eff66b784f0a28db99a11264040e219b3caa64abf19dad5ed7523a9ad
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 2aae771eff66b784f0a28db99a11264040e219b3caa64abf19dad5ed7523a9ad
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:35]:
```suggestion
  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
```
These are opposites of each other, so it seems like they just got swapped.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   64A2A157CD917B6CAF0996ABEDBB32C3291E4B620E38F923FDA057014F5C1FDF
 Expected hash: 2ab7ed6ed297682ec3f10b8b69df969fcc0760ab30ba3f63eccc78e819cc1525
 Specified hash doesn't match.
 Actual hash:   0D320E9FE445EAB017CABF91F8147900B33A620599F92E703F2AE14595675800
 Expected hash: 2aae771eff66b784f0a28db99a11264040e219b3caa64abf19dad5ed7523a9ad
 Specified hash doesn't match.
 Actual hash:   0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
 Expected hash: d79d0c766bad4b87ee029e13ba351565e6b1fe362519a8ba4fae4c4d02ab07bb

(Automated response - build 542.)

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-arm64-nightly.exe
-  InstallerSha256: bba0aed13740573d65657391f91b1b17460866f8aed6030d67c8b6d0dcba2466
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: 2ab7ed6ed297682ec3f10b8b69df969fcc0760ab30ba3f63eccc78e819cc1525
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: 2ab7ed6ed297682ec3f10b8b69df969fcc0760ab30ba3f63eccc78e819cc1525
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:47]:
```suggestion
  InstallerSha256: 64A2A157CD917B6CAF0996ABEDBB32C3291E4B620E38F923FDA057014F5C1FDF
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: arm64
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
  InstallerSha256: 64A2A157CD917B6CAF0996ABEDBB32C3291E4B620E38F923FDA057014F5C1FDF
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 4]
```diff
@@ -1,25 +1,51 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x64-nightly.exe
-  InstallerSha256: c97a9e38345798d522ac4b86f705dfa7d754eb7a74246f29bf03d60b12b3d67c
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 2AAE771EFF66B784F0A28DB99A11264040E219B3CAA64ABF19DAD5ED7523A9AD
+  InstallerSwitches:
+    Custom: /CURRENTUSER
 - Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 2AAE771EFF66B784F0A28DB99A11264040E219B3CAA64ABF19DAD5ED7523A9AD
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:23]:
```suggestion
  InstallerSha256: 0D320E9FE445EAB017CABF91F8147900B33A620599F92E703F2AE14595675800
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
  InstallerSha256: 0D320E9FE445EAB017CABF91F8147900B33A620599F92E703F2AE14595675800
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 5]
```diff
@@ -1,25 +1,51 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:35]:
```suggestion
  InstallerSha256: 0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
  InstallerSha256: 0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
```

**Comment by Exorcism0666** [comment]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   C5D7657E45D4BFF3EA410503D34892AAB06506151FDB937FD6189A8C4CCFB7C3
 Expected hash: 64a2a157cd917b6caf0996abedbb32c3291e4b620e38f923fda057014f5c1fdf
 Specified hash doesn't match.
 Actual hash:   924822B425C80421F441DF5946361F1BBB0181218FFF36B799BC2FD952955CC9
 Expected hash: 0d320e9fe445eab017cabf91f8147900b33a620599f92e703f2ae14595675800
 Specified hash doesn't match.
 Actual hash:   7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
 Expected hash: 0c7cf9d764ea934a9cfd3da9ec7231844545367b0aecad04fa32e1fd4d575ca9

(Automated response - build 549.)

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 6]
```diff
@@ -1,25 +1,54 @@
...
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
+  InstallerSwitches:
+    Custom: /ALLUSERS
 - Architecture: arm64
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-arm64-nightly.exe
-  InstallerSha256: bba0aed13740573d65657391f91b1b17460866f8aed6030d67c8b6d0dcba2466
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: 64A2A157CD917B6CAF0996ABEDBB32C3291E4B620E38F923FDA057014F5C1FDF
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:44]:
```suggestion
  InstallerSha256: C5D7657E45D4BFF3EA410503D34892AAB06506151FDB937FD6189A8C4CCFB7C3
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 7]
```diff
@@ -1,25 +1,54 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: 64A2A157CD917B6CAF0996ABEDBB32C3291E4B620E38F923FDA057014F5C1FDF
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:50]:
```suggestion
  InstallerSha256: C5D7657E45D4BFF3EA410503D34892AAB06506151FDB937FD6189A8C4CCFB7C3
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 8]
```diff
@@ -1,25 +1,54 @@
...
-ReleaseDate: 2023-12-21
+ReleaseDate: 2024-01-05
 Installers:
-- Architecture: x64
+- Architecture: x86
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x64-nightly.exe
-  InstallerSha256: c97a9e38345798d522ac4b86f705dfa7d754eb7a74246f29bf03d60b12b3d67c
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 0D320E9FE445EAB017CABF91F8147900B33A620599F92E703F2AE14595675800
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:20]:
```suggestion
  InstallerSha256: 924822B425C80421F441DF5946361F1BBB0181218FFF36B799BC2FD952955CC9
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 9]
```diff
@@ -1,25 +1,54 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
 - Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 0D320E9FE445EAB017CABF91F8147900B33A620599F92E703F2AE14595675800
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:26]:
```suggestion
  InstallerSha256: 924822B425C80421F441DF5946361F1BBB0181218FFF36B799BC2FD952955CC9
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 10]
```diff
@@ -1,25 +1,54 @@
...
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:32]:
```suggestion
  InstallerSha256: 7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 11]
```diff
@@ -1,25 +1,54 @@
...
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 0C7CF9D764EA934A9CFD3DA9EC7231844545367B0AECAD04FA32E1FD4D575CA9
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:38]:
```suggestion
  InstallerSha256: 7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-01-10 22:17:32.336 [FAIL] Installer failed security check. Url: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe Result: 0x80004005

(Automated response - build 557.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-01-10 22:17:32.336 [FAIL] Installer failed security check. Url: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe Result: 0x80004005

(Automated response - build 557.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 557.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 570.)

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 12]
```diff
@@ -1,25 +1,54 @@
...
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
 Commands:
 - komac
-ReleaseDate: 2023-12-21
+ReleaseDate: 2024-01-05
 Installers:
-- Architecture: x64
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:17]:
```suggestion
- Architecture: x64
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 13]
```diff
@@ -1,25 +1,54 @@
...
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x64-nightly.exe
-  InstallerSha256: c97a9e38345798d522ac4b86f705dfa7d754eb7a74246f29bf03d60b12b3d67c
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 924822B425C80421F441DF5946361F1BBB0181218FFF36B799BC2FD952955CC9
+  InstallerSwitches:
+    Custom: /CURRENTUSER
 - Architecture: x86
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:23]:
```suggestion
- Architecture: x64
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 14]
```diff
@@ -1,25 +1,54 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: 924822B425C80421F441DF5946361F1BBB0181218FFF36B799BC2FD952955CC9
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:29]:
```suggestion
- Architecture: x86
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 15]
```diff
@@ -1,25 +1,54 @@
...
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:35]:
```suggestion
- Architecture: x86
```

**Comment by stephengillie** [comment]
> Actual hash:   C0F1A866E9A75FAD2C8DF7850BC0C677E3187D9E1699DE1473E5B247AAA661ED
Expected hash: 7438b658cbca8276621d35a51558d7c6792b99c6852ca30d55003ca508b3a6c5
Actual hash:   7FD66B79AB830EB21CDA99122D9EB2EE10A5674559B0B9BDF22341016B9EB2E0
Expected hash: c5d7657e45d4bff3ea410503d34892aab06506151fdb937fd6189a8c4ccfb7c3
Actual hash:   DEBD059A3FBF46D946C9902E46B665FF91C2E67BFEB61E3AB058097DC3FABCA9
Expected hash: 924822b425c80421f441df5946361f1bbb0181218fff36b799bc2fd952955cc9

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 16]
```diff
@@ -1,25 +1,54 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 7438B658CBCA8276621D35A51558D7C6792B99C6852CA30D55003CA508B3A6C5
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:38]:
```suggestion
  InstallerSha256: C0F1A866E9A75FAD2C8DF7850BC0C677E3187D9E1699DE1473E5B247AAA661ED
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
  InstallerSha256: C0F1A866E9A75FAD2C8DF7850BC0C677E3187D9E1699DE1473E5B247AAA661ED
```

File: manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml [hunk 17]
```diff
@@ -1,25 +1,54 @@
...
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-arm64-nightly.exe
-  InstallerSha256: bba0aed13740573d65657391f91b1b17460866f8aed6030d67c8b6d0dcba2466
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: C5D7657E45D4BFF3EA410503D34892AAB06506151FDB937FD6189A8C4CCFB7C3
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
+  InstallerSha256: C5D7657E45D4BFF3EA410503D34892AAB06506151FDB937FD6189A8C4CCFB7C3
```
Comment (stephengillie) [inline manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml:50]:
```suggestion
  InstallerSha256: 7FD66B79AB830EB21CDA99122D9EB2EE10A5674559B0B9BDF22341016B9EB2E0
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: arm64
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-arm64.exe
  InstallerSha256: 7FD66B79AB830EB21CDA99122D9EB2EE10A5674559B0B9BDF22341016B9EB2E0
```

**Pull request closed without merging**